test/Scripts.Tests/test-pack-contents.ps1

```diff
@@ -0,0 +1,47 @@
+# Verifies the contents of the UPM package against a snapshot file
+# 'dotnet build' and 'pack.ps1' must have already been run
+
+# To accept a new snapshot file, run 'pwsh ./test.ps1 accept'
```

Comment:
```suggestion
# To accept a new snapshot file, run 'pwsh ./test/Scripts.Tests/test-pack-contents.ps1 accept' 
```

Reply:
The path is below this line so I avoided a copy paste here